components/Footer/styles.css

```diff
@@ -33,11 +33,137 @@
   font-size: 1em;
 }
 
+.PreFooter {
+    height: 135px;
+    display: flex;
+    margin-bottom: 10px;
+    display: -ms-flexbox;
+    -ms-flex-align: center;
+    font-family: Lato;
+    color: white;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    font-family: Lato;
+    margin-top: 10px;
+}
+
+.GetInTouch {
+    width: 22%;
+    background-color: #f36505;
+    margin-right: 10px;
+    font-size: 33px;
+    min-width: 250px;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+.GetInTouch a {
+    text-decoration: none;
+    color: white;
+}
+
+.StayInformed {
+    width: 78%;
+    background-color: #f36505;
+    height: 100%;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    padding: 15px 3% 15px 3%;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+.s1 {
+    width: 24%;
+    margin-right: 2vw;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    min-width: 250px;
+}
+
+.s2{
+  width: 33%;
+  min-width: 172px
+}
+
+.s3{
+  width: 33%;
+}
+
+p.title {
+    font-size: 33px;
+    font-weight: inherit;
+    line-height: 0px;
+    margin-bottom: 26px;
+}
+
+p.subtitle {
+    line-height: 23px;
+    font-weight: lighter;
+    margin-top: 5px;
+}
+
+input[type="text"] {
```

Comment:
use a class here to style the input so these styles dont carry over to any other inputs. Something like `PreFooter-input`